### PR TITLE
refactor(reports): rename stock_sheet

### LINKED ISSUE #4386: Add SUMs to the bottom of the fiche de stock
The fiche de stock report currently lack sums for the total value of the entry and exit columns.  Should we add sums on them?  They would show up here:

image

@mbayopanda, what do you think of this proposal?  I am suspicious that this wasn't done for a reason.

### PR DESCRIPTION
**Pull request by jniles**
Adds in the proper name of the Stock Sheet (Fiche de Stock) in English and updates all paths to reflect this.  Previously, we had multiple confusing names (inventory_file on the client, inventory_report on server).  We also have another report called `inventories_report` that exacerbated the confusion.  Now, it's all `stock_sheet`.

Also updates the parameter parsing to in line with most other reports by removing JSON.parse() from the URL and making the URL readable.

Finally, we have added SUMs to the bottom of the stock_sheet.

Closes #4386.

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda, can I get a review?

File: client/src/modules/reports/generate/stock_sheet/stock_sheet.config.js [hunk 1]
```diff
@@ -46,27 +49,16 @@ function InventoryFileConfigController($sce, Notify, SavedReports, AppCache, rep
...
-      params,
-      lang : Languages.key,
-    };
-
-    vm.reportDetails = options;
+    options.dateTo = moment(options.dateTo).format('YYYY-MM-DD');
+    options.lang = Languages.key;
 
-    return SavedReports.requestPreview(reportUrl, reportData.id, angular.copy(vm.reportDetails))
+    return SavedReports.requestPreview(reportUrl, reportData.id, options)
```
Comment (mbayopanda) [inline client/src/modules/reports/generate/stock_sheet/stock_sheet.config.js:61]: 👍

File: server/config/routes.js [hunk 2]
```diff
@@ -835,7 +835,7 @@ exports.configure = function configure(app) {
   app.get('/reports/stock/lots', stockReports.stockLotsReport);
   app.get('/reports/stock/movements', stockReports.stockMovementsReport);
   app.get('/reports/stock/inventories', stockReports.stockInventoriesReport);
-  app.get('/reports/stock/inventory', stockReports.stockInventoryReport);
+  app.get('/reports/stock/sheet', stockReports.stockSheetReport);
```
Comment (mbayopanda) [inline server/config/routes.js:838]: It is more meaningful 👍

**Review by mbayopanda — COMMENTED** [review]
This is a good work, is it also possible to rename the file `stock_inventory.report.handlebars` to `stock_sheet.report.handlebars` ?

**Comment by jniles** [comment]
@mbayopanda thanks for the review!

Certainly, I missed that.  I'll make the change.

**Comment by jniles** [comment]
@mbayopanda all done.

**Pull request merged**